tsconfig.vite-config.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
Just a note: The formatting here should get fixed once you set up Prettier.

Reply:
Sure